docs/jupyter/notebooks/assets.ipynb

```diff
@@ -45,7 +45,7 @@
     "\n",
     "This pandas pipe function will work when there are the following columns in the DataFrame:\n",
     "- A column name that contains `host_id`\n",
-    "- A column name that contains `tenant.id` (or `tenant_id`)\n",
+    "- A column name that contains `tenant.id`\n",
```

Comment:
The dual column lookup is still correct.  Events still use `tenant_id`, it was alerts that deprecated the `tenant_id` string in favor of a Tenant object.

Event schemas can be verified using the following snippet, change the RuleEventType to one of the options [here](https://github.com/secureworks/taegis-sdk-python/blob/main/taegis_sdk_python/services/rules/types.py#L21):

```python
from taegis_sdk_python import GraphQLService
from taegis_sdk_python.services.rules.types import RuleEventType

service = GraphQLService()

results = service.rules.query.filter_keys(RuleEventType.PROCESS)
print(results)
```

https://github.com/secureworks/taegis-magic/blob/main/taegis_magic/pandas/assets.py#L124

https://github.com/secureworks/taegis-sdk-python/blob/main/taegis_sdk_python/services/alerts/types.py#L3111